polaris-core/src/test/java/org/apache/polaris/core/storage/StorageUtilTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.storage;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class StorageUtilTest {
+
+  @Test
+  public void testEmptyString() {
+    Assertions.assertThat(StorageUtil.getBucket("")).isNull();
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"s3", "gcs", "azure"})
```

Comment:
nit: do you think here values should be inline with iceberg supported [scheme](https://github.com/apache/iceberg/blob/f7c6d57a03282516c6f988fa23a809416cd765b1/core/src/main/java/org/apache/iceberg/io/ResolvingFileIO.java#L59) ? something like
`s3, s3a, s3n, gs, abfs, abfss`